README.md

```diff
@@ -0,0 +1,77 @@
+# Flatpak for openMSX
```

Comment:
My usual comment about README's in Flathub repos is that any user-facing information is likely going to be ignored, and things related to development is duplication of the documentation, but you feel strongly about keeping it here, I won't object.

Reply:
Yeah, I agree that there is little change that users will find it this way, but the description is just a copy/paste from the XML, and the rest is just a template that I modified a bit, so it was pretty quick to put it together. And I have seen various other flathub repos that have it.